website/settings/local-dist.py

```diff
@@ -32,9 +39,15 @@
 ENABLE_EMAIL_SUBSCRIPTIONS = False
 
 # Session
-OSF_COOKIE_DOMAIN = None
 COOKIE_NAME = 'osf'
-SECRET_KEY = "CHANGEME"
+OSF_COOKIE_DOMAIN = None
+SECRET_KEY = 'CHANGEME'
+SESSION_COOKIE_SECURE = SECURE_MODE
+SESSION_COOKIE_HTTPONLY = False
+
+# certificate for local SECURE_MODE development
+OSF_SERVER_KEY = 'PATH_TO_LOCAL_PRIVATE_KEY'
```

Comment:
Are these paths to a file?

If so, can you please change the default value to `None`, and make "path to local private key" a source code comment instead? 

Explicitly indicating that no default value is provided can help avoid confusing error messages that happen when some placeholder text gets treated like an actual value.